liquibase-cli/src/main/java/liquibase/integration/commandline/LiquibaseCommandLine.java

```diff
@@ -360,6 +366,53 @@ public int execute(String[] args) {
         }
     }
 
+    protected void enableMonitoring() {
+        final liquibase.logging.Logger log = Scope.getCurrentScope().getLog(getClass());
+
+        try {
+            final String monitorPerformanceValue = LiquibaseCommandLineConfiguration.MONITOR_PERFORMANCE.getCurrentValue();
+            if (monitorPerformanceValue == null || monitorPerformanceValue.equalsIgnoreCase("false")) {
+                log.fine("Performance monitoring disabled");
+                return;
+            }
+
+            final String version = System.getProperty("java.version");
+            final String[] splitVersion = version.split("\\.", 2);
+            if (Integer.parseInt(splitVersion[0]) < 11) {
+                Scope.getCurrentScope().getUI().sendMessage("Performance monitoring requires Java 11 or greater. Version " + version + " is not supported.");
+                return;
+            }
+
+
+            String filename = monitorPerformanceValue;
+            if (filename.equalsIgnoreCase("true")) {
+                filename = "liquibase-" + new ISODateFormat().format(new Date()).replaceAll("\\W", "_") + ".jfr";
+            }
+            if (!filename.endsWith(".jfr")) {
+                filename = filename + ".jfr";
+            }
+
+            final Class<?> configurationClass = Class.forName("jdk.jfr.Configuration");
+            final Class<?> recordingClass = Class.forName("jdk.jfr.Recording");
```

Comment:
This could be a good place to catch class not found exception here and print out some helpful messages here. For example, letting the user know that they need to be running liquibase with jdk instead of jre.

Reply:
We're catching whatever exceptions happen and then logging that out, including class not found exceptions. With Java 11+ there isn't really JREs anymore so assuming something like that in the output gets harder to do. I think the existing message about "can't enable logging because of e.getMessage()" is probably as a good as we can do